### PR TITLE
Update privacy policy texts (EXPOSUREAPP-8334)

### PR DESCRIPTION
**Pull request by SamuraiKek**

### CODE REVIEW

File: Corona-Warn-App/src/main/assets/privacy_en.html [hunk 1]
```diff
@@ -1,1581 +1,1088 @@
-
 <p>
-    This privacy notice explains how your data is processed and what data
-    protection rights you have when using the German Federal Government’s
-    official coronavirus app, the Corona-Warn-App.
+    In dieser Datenschutzerklärung erfahren Sie, wie Ihre Daten verarbeitet
+    werden und welche Datenschutzrechte Sie haben, wenn Sie die offizielle
+    Corona-Warn-App der deutschen Bundesregierung nutzen.
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/assets/privacy_en.html:4]: this should be the English file

File: Corona-Warn-App/src/main/assets/privacy_tr.html [hunk 2]
```diff
@@ -1,1506 +1,1088 @@
-
 <p>
-    Bu Veri Gizliliği Beyanında, Almanya Federal Hükümetinin resmi Corona-Warn-App’ını
-    kullandığınızda, verilerinizin nasıl işleneceğini ve hangi veri koruma haklarına sahip
-    olduğunuzu öğreneceksiniz.
+    In dieser Datenschutzerklärung erfahren Sie, wie Ihre Daten verarbeitet
+    werden und welche Datenschutzrechte Sie haben, wenn Sie die offizielle
+    Corona-Warn-App der deutschen Bundesregierung nutzen.
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/assets/privacy_tr.html:4]: this the Turkish file

**Comment by mtwalli** [comment]
Please use the right file / language ? they are attached to the ticket

**Review by ralfgehrer — CHANGES_REQUESTED** [review]
language mix up.

File: Corona-Warn-App/src/main/assets/privacy_en.html (hunk 1 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/assets/privacy_en.html:4]: Rookie mystake...

**Review by mtwalli — APPROVED** [review]

**Review by ralfgehrer — APPROVED** [review]

**Pull request merged**